continuous-pipe.yml.erb

```diff
@@ -220,3 +248,17 @@ tasks:
               limits:
                 cpu: 1
                 memory: 1G
+
+  migration:
+    run:
+      image:
+        from_service: web
+
+      commands:
+        - container setup
```

Comment:
can we just have the ubuntu base image do_setup call do_migrate with a deprecation message?